driver/src/driver/stablex_driver.rs

```diff
@@ -62,14 +62,30 @@ impl<'a> StableXDriver<'a> {
             info!("No orders in batch {}", batch_to_solve);
             Solution::trivial(0)
         } else {
-            let price_finder_result = self.price_finder.find_prices(&orders, &account_state);
+            let solver_time_limit_result =
+                self.orderbook_reader.get_auction_remaining_solution_time();
             self.metrics
-                .auction_solution_computed(batch_to_solve, &orders, &price_finder_result);
-
-            let solution = price_finder_result?;
-            info!("Computed solution: {:?}", &solution);
-
-            solution
+                .auction_solver_time_limit(&solver_time_limit_result);
+            let solver_time_limit = solver_time_limit_result?;
+
+            if solver_time_limit.as_secs() == 0 {
+                info!("Not enough time remaning in batch to compute solution");
+                Solution::trivial(0)
```

Comment:
I think this should not log a skipped event, because we actually don't process the batch. And so as far as our "success" metric is concerned we did not succeed (and it was kind of our fault for being late)